### ISSUE TITLE
Ability to hide from menu bar

### ISSUE THREAD

**Issue by frozenpandaman** [Pyroh/Fluor#66]
A feature allowing the application to be hidden from the menu bar and just run in the background would be great.

**Comment by YunfangHou** [comment]
You may be interested in Bartender.

**Comment by frozenpandaman** [comment]
I'm aware of it! But it'd be nice to have this as a native option.

**Comment by YunfangHou** [comment]
> I'm aware of it! But it'd be nice to have this as a native option.

You are right. I appreciate the apps with the function of hide menu icon. It's something like the art and philosophy of app designing.

**Comment by arjunahmtp** [comment]
> A feature allowing the application to be hidden from the menu bar and just run in the background would be great.

#42 therequest is its already there and has been picked as an enhancement but I don't think its been made available as yet
  #42
